tests/backup/backup_resiliency_test.go

```diff
@@ -750,12 +771,13 @@ var _ = Describe("{CancelAllRunningBackupJobs}", func() {
 					}
 					return "", false, nil
 				}
-				_, err = task.DoRetryWithTimeout(backupJobCancelStatus, maxWaitPeriodForBackupJobCancellation*time.Minute, backupJobCancellationRetryTime*time.Second)
+				_, err = task.DoRetryWithTimeout(backupJobCancelStatus, backupDeleteTimeout*time.Minute, backupDeleteRetryTime*time.Second)
 				if err != nil {
 					adminBackups, error1 := GetAllBackupsAdmin()
-					log.Infof("The list of backups still present after backup cancellation and wait of 10 minutes is %v,Error:%v", adminBackups, err)
 					log.FailOnError(error1, "Getting the list of backups after backup cancellation and wait of 10 minutes")
+					log.Infof("The list of backups still present after backup cancellation and wait of 10 minutes is %v,Error:%v", adminBackups, err)
```

Comment:
The log need to be edited, The wait minute is not static.

Reply:
corrected